mmpose/core/evaluation/top_down_eval.py

```diff
@@ -574,3 +611,13 @@ def keypoints_from_heatmaps(heatmaps,
         maxvals = maxvals / 255.0 + 0.5
 
     return preds, maxvals
+
+
+def multilabel_classification_accuracy(pred, gt, mask, thr=0.5):
+    valid = (mask > 0).min(axis=1) if mask.ndim == 2 else (mask > 0)
+    pred, gt = pred[valid], gt[valid]
+    if pred.shape[0] == 0:
+        acc = 0  # when no sample is with gt labels, set acc to 0.
+    else:
+        acc = (((pred - thr) * (gt - thr)).min(axis=1) > 0).mean()
```

Comment:
What's the meaning of the axis 1?

Reply:
The classification of a sample is regarded as correct only if it's correct for all labels. 
I changed it to a way easier to understand.